R/survey_statistics.r

```diff
@@ -334,12 +334,20 @@ get_var_est <- function(stat, vartype, var_names = "", grps = "",
       se
     } else if (vvv == "ci") {
       if (!quantile) {
-        ci <- data.frame(matrix(confint(stat, level = level),
-                                ncol = 2 * out_width))
+        if (length(level)==1) {
+          ci <- data.frame(matrix(confint(stat, level = level),
+                                  ncol = 2 * out_width))
+          names(ci) <- c(paste0(var_names, "_low"), paste0(var_names, "_upp"))
+        } else {
+          lci <- lapply(level, function(x) {as.data.frame(confint(stat,level = x))})
+          ci <- dplyr::bind_cols(lci)
+          names(ci) <- paste0(var_names,"_", names(ci))
```

Comment:
I don't love the variable names. Currently example: `api99_mn_12.5 %` and `api99_mn_2.5 %`

My first preference is  `api99_mn_low75` and `api99_mn_low95`. This gets around decimals (unless the user specifies a decimal in the level, which seems unlikely).

My next preference would be to take from survey_quantile, and replace with: `api99_mn_125` and `api99_mn_025`. I know that the `125` bit is confusing, but all numbers technically are differentiated because there's a leading 0 for numbers less than 10. See line 139 of this file for the details.

I could be convinced to: `api99_mn_12.5` and `api99_mn_97.5`, but I definitely think the space and % sign need to go (" %").